democluster: shorten the scheduled jobs initial delay

otan opened the pull request:
Release note (cli change): cockroach demo will now begin processing
scheduled jobs after 15s, instead of the 2-5min in a production
environment.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/70242)
<!-- Reviewable:end -->

otan committed:
democluster: shorten the scheduled jobs initial delay

Release note (cli change): cockroach demo will now begin processing
scheduled jobs after 15s, instead of the 2-5min in a production
environment.

knz reviewed (approved):
> Reviewed 3 of 3 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/70242)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @dt)



<!-- Sent from Reviewable.io -->

dt reviewed (approved):
Thanks!

otan commented:
bors r=dt,knz
